scripts/compose/default.env

```diff
@@ -1,6 +1,6 @@
-HOPRD_ENVIRONMENT_ID=master-staging
-HOPRD_DOCKER_IMAGE=gcr.io/hoprassociation/hoprd:master-staging
+HOPRD_ENVIRONMENT=monte_rosa
```

Comment:
I see everything fine, for the sake of simplicity I would rename this file to `.env` so the user does not need to explicitly name it in the commandline.
This is not a blocker

Reply:
Renamed file to .env